packages/visual-editor/src/components/Draggable/canvasToolsUtils.ts

```diff
@@ -0,0 +1,167 @@
+import type {
+  ComponentDefinition,
+  CompositionComponentNode,
+  CompositionComponentPropValue,
+  CompositionDataSource,
+  CompositionUnboundValues,
+  CompositionVariableValueType,
+  Link,
+} from '@contentful/experience-builder-core/types';
+import { ASSEMBLY_DEFAULT_CATEGORY } from '@contentful/experience-builder-core/constants';
+
+export type ExperienceDataSource = CompositionDataSource;
+export type ExperienceUnboundValues = CompositionUnboundValues;
+export type ExperienceProperty = CompositionComponentPropValue;
+export type ExperiencePropertyLiteral = CompositionVariableValueType;
```

Comment:
These type aliases seem unnecessary to me.  We could probably stick with the Composition-named types or rename them at the source if the names need to change to start with Experience.